setup.py

```diff
@@ -5,9 +5,9 @@
 install_requires = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.8',
     'django-appconf',
-    'django-cms>=3.0',
+    'django-cms>=3.5',
```

Comment:
we still support < 3.5

Reply:
Is it somehow possible to make these changes optional? So only activate them if django-cms > 3.4?